frontends/web/src/routes/buy/components/exchangeselectionradio.tsx

```diff
@@ -43,8 +44,8 @@ const Deal = ({ deal }: { deal: ExchangeDealWithBestDeal }) => {
     <div className={style.paymentMethodContainer}>
       <p className={style.paymentMethodName}>{getPaymentMethodName(deal.payment)}</p>
       <div>
-        <span>{deal.isBestDeal && t('buy.exchange.bestDeal')}</span>
-        <span>{deal.isFast && t('buy.exchange.fast')}</span>
+        <span>{deal.isBestDeal && <BestDeal />}</span>
+        <span>{deal.isFast && <Fast />}</span>
```

Comment:
div's inside span's is technically not valid html.
does it need the wrapping `<span>`?

Reply:
oh good catch, yeah. no need for span.